plugins/default/lib/disk_usage.py

```diff
@@ -53,6 +62,10 @@ def analize(pod, disks, warning_threshold, critical_threshold):
 
 
 def report(results):
+    # REVIEW: is nagios.UNKNOWN the best here?
```

Comment:
This is handling the case when we didn't get any lines from `df`.

@jessesarn what would you suggest we do in this case?